metis-enrichment/metis-enrichment-rest/src/main/java/eu/europeana/enrichment/rest/EnrichmentController.java

```diff
@@ -80,7 +80,7 @@ public EnrichmentResultList search(
   @ApiResponses(value = {@ApiResponse(code = 400, message = "Error processing the result")})
   public EnrichmentBase equivalence(@ApiParam("uri") @RequestParam("uri") String uri) {
     return enrichmentService
-        .enrichByEquivalenceValues(new ReferenceValue(uri, Collections.emptySet()));
+        .enrichByEquivalenceValues(new ReferenceValue(uri, Collections.emptySet())).get(0);
```

Comment:
This is right. But perhaps put a comment saying that we expect only one, and that perhaps in the future we should handle this better.

Reply:
Good point, will do that